apps/frontend/components/NavBar/NavMenuContent.tsx

```diff
@@ -28,6 +28,7 @@ export const NavMenuContent = (): JSX.Element => {
   const isAdmin = useIsSignedInUserAdmin();
 
   const handleAdminClick = () => {
+    // TODO - Use apiClient.baseUrl?
```

Comment:
We're already getting the env in `apiClient.baseUrl`

Reply:
I think here the comment is about not calling env() but using apiClient.baseurl. But I think it is ok as it